src/content/apis/dictionary/dictionaryapi.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Free Dictionary API
+description: Easily find the meaning of the words using this free api.
+categories:
+  - dictionary
```

Comment:
Please change the category to `miscs`.
And also move the file to the miscs directory.